### ISSUE TITLE
Custom mode doesn't seem to work anymore

### ISSUE THREAD

**Issue by vrad-exe** [ray-lothian/UserAgent-Switcher#230]
I use this extension's "custom" mode in Firefox to override the user agent sent to Gmail to be Chrome, since Google arbitrarily locks all other browsers out of offline sync functionality despite it being supported fine. However, as of somewhat recently this randomly stopped working, with Gmail now refusing to enable offline mode unless I open the extension and select a Chrome user agent manually. I haven't touched the extension settings at all (except changing the Chrome UA to a newer version in an attempt to fix the issue), so I have to assume this broke with an extension or browser update.

This is the specific custom json I use:
```
{
  "google.com": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36 GLS/100.10.9939.100",
  "microsoft.com, office.com, skype.com, bing.com": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36 Edg/114.0.1788.0"
}
```

If I copy and paste the exact same Chrome UA into the extension popup and apply it manually, it works fine

**Comment by Leah96xxx** [comment]
For me it seems to only work after opening the extension's settings. This needs to be done every time the browser is restarted.

**Comment by sofiedotcafe** [comment]
> For me it seems to only work after opening the extension's settings. This needs to be done every time the browser is restarted. Some times it won't work until I click Save or until I do a ctrl+f5 on the tab with the wrong UA

Same, made an duplicate #240, closing that!

**Comment by BTubbs200** [comment]
Yep, I only have one specific website that I need my user agent switched on, and after following the correct syntax for inputting the website and useragent string into custom mode and saving, nothing happens when visiting the site. Very frustrating.

**Comment by sofiedotcafe** [comment]
> Yep, I only have one specific website that I need my user agent switched on, and after following the correct syntax for inputting the website and useragent string into custom mode and saving, nothing happens. Very frustrating. 

Try https://github.com/ray-lothian/UserAgent-Switcher/issues/240#issue-2858537070, bit of an workaround but should work.

I have even found the UA customization option being more usable in the "try in mobile" mode in the dev-tools than this extension.

**Comment by vrad-exe** [comment]
This is still broken.

**Comment by ray-lothian** [comment]
What is the color of the toolbar button when you use the above custom JSON? Do you get any error on the console? What is the extension version

**Comment by vrad-exe** [comment]
When using the provided JSON on Gmail (`mail.google.com`), the toolbar icon appears half orange and half blue. There are a few user agent related messages appearing in the console which all look like this just with different URLs:
```
[user-agent leaked] using async method https://mail.google.com/mail/u/0/#inbox
```
